src/ComparedQueryable/ComparedQueryable.csproj

```diff
@@ -40,15 +40,22 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="NativeQueryable\CachedReflection.cs" />
+    <Compile Include="EnumerableExecutor[TCollectionElement,TComparison].cs" />
+    <Compile Include="EnumerableQuery[TCollectionElement,TComparison].cs" />
+    <Compile Include="EnumerableRewriter[TComparison].cs" />
     <Compile Include="NativeQueryable\EnumerableExecutor.cs" />
     <Compile Include="NativeQueryable\EnumerableQuery.cs" />
     <Compile Include="NativeQueryable\EnumerableRewriter.cs" />
     <Compile Include="NativeQueryable\Error.cs" />
-    <Compile Include="NativeQueryable\Queryable.cs" />
-    <Compile Include="NativeQueryable\Strings.cs" />
+    <Compile Include="QueryableExtensions.cs" />
     <Compile Include="NativeQueryable\TypeHelper.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\NaturalSort\NaturalSort.csproj">

```

Comment:
Ideally, we would first put out the nuget package for the NaturalSort algorithm, then import that package, but for now, this helps gets things working.